Server/Plugins/APIDump/Classes/World.lua

```diff
@@ -3119,6 +3119,30 @@ function OnAllChunksAvailable()</pre> All return values from the callbacks are i
 				},
 				Notes = "Returns whether coloured chat prefixes are prepended to chat messages or the entire message is simply coloured.",
 			},
+			SpawnArmorStand =
+			{
+				{
+					Params =
+					{
+						{
+							Name = "Position",
+							Type = "Vector3d",
+						},
+						{
+							Name = "Yaw",
+							Type = "number",
+						},
+					},
+					Returns =
+					{
+						{
+							Name = "EntityID",
+							Type = "number",
+						},
+					},
+					Desc = "Spawns an armor stand at the given coordinates and with the given orientation in degrees from south, going clockwises. Returns the UniqueID of the spawned armor stand, or {{cEntity#INVALID_ID|cEntity#INVALID_ID}} on failure.",
```

Comment:
Please don't use `south`, we prefer to name the directions according to the axes: `X+`, `Z-` etc. The Mojang definition of "south" has changed at least once in Minecraft history.